.github/deploy.sh

```diff
@@ -25,4 +25,44 @@ then
   git commit -m "[auto] [ci skip] Generate APPSTORE"
 fi
 
+if [[ $status == *"LATEST.md"* ]]
+then
+  git add APPSTORE.md
+  git commit -m "[auto] [ci skip] Generate LATEST"
+fi
+
+git push --quiet "https://${GH_TOKEN}@github.com/dkhamsing/open-source-ios-apps" master:master > /dev/null 2>&1
```

Comment:
hey @vpeschenkov do we need all these changes??

Reply:
@dkhamsing my bad; didn't pay attention when moving this code